main/res/values/strings.xml

```diff
@@ -580,6 +583,7 @@
     <string name="settings_open_geokretymap_website">Open GeoKretyMap.org website</string>
     <string name="settings_settings">Settings</string>
     <string name="settings_information">Information</string>
+    <string name="settings_enhancement">Enhancement</string>
```

Comment:
I'm not even sure what this is referring to, but I believe the term might be wrong, as I can't imagine this to appear anywhere in a settings screen. Is this referring to "Extensions", "Advanced settings", or something similar?